src/lib/types.ts

```diff
@@ -0,0 +1,10 @@
+export interface Product {
+    id: string;
+    name: string;
+    description: string;
+    price: string;
+  }
+
+export interface ProductTableProps {
```

Comment:
Probably we need not define them separately, we can define them in respective components/services

Reply:
ok ,sure